src/current/v22.2/row-level-ttl.md

```diff
@@ -140,7 +140,7 @@ The settings that control the behavior of Row-Level TTL are provided using [stor
 | `ttl` <a name="param-ttl"></a>                                             | Signifies if a TTL is active. Automatically set.                                                                                                                                                                                                                                                                                                                                                                                        | N/A                                 |
 | `ttl_select_batch_size`                                                    | How many rows to [select](select-clause.html) at one time during the row expiration check. Default: 500. Minimum: 1.                                                                                                                                                                                                                                                                                                                    | `sql.ttl.default_select_batch_size` |
 | `ttl_delete_batch_size` <a name="param-ttl-delete-batch-size"></a>         | How many rows to [delete](delete.html) at a time. Default: 100. Minimum: 1.                                                                                                                                                                                                                                                                                                                                                             | `sql.ttl.default_delete_batch_size` |
-| `ttl_delete_rate_limit`                                                    | Maximum number of rows to be deleted per second (rate limit). Default: 0 (no limit).                                                                                                                                                                                                                                                                                                                                                    | `sql.ttl.default_delete_rate_limit` |
+| `ttl_delete_rate_limit` | Maximum number of rows to be deleted per second (rate limit). Default: 0 (no limit). <br/><br/> Note: The rate limit will vary in practice based on the number of [ranges]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range) and [leaseholders]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-leaseholder). It is applied per leaseholder per table. <br/> For example, if this variable is set to 500, and a table `public.foo` has its ranges spread across 3 leaseholders, then the cluster-wide rate limit for `public.foo` is 500 x 3 = 1500. | `sql.ttl.default_delete_rate_limit` |
```

Comment:
```suggestion
| `ttl_delete_rate_limit` | Maximum number of rows to be deleted per second (rate limit). Default: 0 (no limit). <br/><br/> Note: The rate limit is applied per leaseholder per table. In practice, it will vary based on the number of [ranges]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range) and [leaseholders]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-leaseholder).<br/> For example, if this variable is set to 500, and a table `public.foo` has its ranges spread across 3 leaseholders, then the cluster-wide rate limit for `public.foo` is 500 x 3 = 1500. | `sql.ttl.default_delete_rate_limit` |
```

Reply:
thanks, this is way better.  applied to all versions of this text